etcdIntf/gobuild.toml

```diff
@@ -0,0 +1,23 @@
+[vars]
+  ignoreDirs = [".git", "vendor"]
```

Comment:
shouldn't need this large of a file. just override the testCoverage var is all you want right?

Reply:
done